fastify.js

```diff
@@ -241,6 +227,18 @@ function fastify (options) {
     }
   })
 
+  ;[
+    { f: 'prefix', s: kRoutePrefix },
+    { f: 'validatorCompiler', s: kValidatorCompiler },
+    { f: 'serializerCompiler', s: kSerializerCompiler }
+  ].forEach(({ f, s }) => {
```

Comment:
Can these variables be named with more clarity? It is not obvious what `f` and `s` refer to.